### PR TITLE
Add licensing

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Add MIT licensing to the project. Change was also made in brown-ccv repo

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Center for Computation and Visualization, Brown University
```
Comment (mcmcgrath13) [inline LICENSE:3]: @mirestrepo should this be CCV or RIDDC?

Comment (RobertGemmaJr) [reply LICENSE:3]: Rhode Island Data Discovery Center, right?

Comment (RobertGemmaJr) [reply LICENSE]: I changed it to "Copyright (c) 2021 Rhode Island Data Discovery Center"

Will wait to merge until I get confirmation that RIDDC should own the copyright for the project

Comment (mirestrepo) [reply LICENSE]: I asked Jeff and he asked if we could put both RIDDC and Brown University

**Comment by RobertGemmaJr** [comment]
I added them both  on different lines:
```
MIT License

Copyright (c) 2021 Center for Computation and Visualization, Brown University
Copyright (c) 2021 Rhode Island Data Discovery Center
```

Or should it be
```
MIT License

Copyright (c) 2021 Center for Computation and Visualization, Brown University & Rhode Island Data Discovery Center
```

**Comment by mirestrepo** [comment]
> I added them both on different lines:
> 
> ```
> MIT License
> 
> Copyright (c) 2021 Center for Computation and Visualization, Brown University
> Copyright (c) 2021 Rhode Island Data Discovery Center
> ```
> 
> Or should it be
> 
> ```
> MIT License
> 
> Copyright (c) 2021 Center for Computation and Visualization, Brown University & Rhode Island Data Discovery Center
> ```

Take CCV out...

Copyright (c) 2021 Rhode Island Data Discovery Center
Copyright (c) 2021 Brown University

**Comment by RobertGemmaJr** [comment]
Done! Should be all set now 😄

**Pull request merged**